src/AccessTokenClient.ts

```diff
@@ -41,6 +41,8 @@ export class AccessTokenClient {
       this.assertNonEmptyPreAuthorizedCode(accessTokenRequest);
       this.assertNumericPin(accessTokenRequest, isPinRequired);
       this.assertNonEmptyClientId(accessTokenRequest);
+    } else {
+      throw new Error('Non-pre-authorized flow is not yet supported.');
```

Comment:
Loving you double negatives 😂 
'Only pre-authorized-code flow is supported' ;)

Reply:
pushing fix.